.github/workflows/pre-merge-rust.yml

```diff
@@ -90,3 +90,28 @@ jobs:
       working-directory: ${{ matrix.dir }}
       # NOTE: --all-targets doesn't run doc tests
       run: cargo test --locked --all-targets
+
+  block-manager-rust:
+    runs-on:
+      group: Fastchecker
```

Comment:
For my own education, I'm curious about when we should use this fastchecker group vs. default behavior from Harrison/Anant